iaso/models/entity.py

```diff
@@ -57,6 +67,66 @@ def as_dict(self):
         }
 
 
+class InvalidLimitDateError(ValidationError):
+    pass
+
+
+class InvalidJsonContentError(ValidationError):
+    pass
+
+
+class EntityQuerySet(models.QuerySet):
+    def filter_for_mobile_entity(self, limit_date=None, json_content=None):
+        if limit_date:
+            try:
+                self = self.filter(instances__updated_at__gte=limit_date)
+            except ValidationError:
+                raise InvalidLimitDateError(f"Invalid limit date {limit_date}")
```

Comment:
I don't understand why this would raise an error. The client passes the date to indicate the last time they synced. If there were no changes since, the array returned should just be empty

Reply:
Well this will not raise if there are no instance but if the limit_date has an invalid format , if no instance it will just return an empty array of instances ...